### DISCUSSION TITLE
Please consider Unlimited Sites pricing

### DISCUSSION THREAD

**Discussion by weavermedia** [avo-hq/avo#478] [Feature requests]
Please consider adding an 'unlimited sites' purchase option to your current per-site model. Other similar projects did the same, like the Jumpstart Pro template for Rails apps. For early adopters it was $149 per site or $449 for unlimited sites.

The per-site model may be something of a hinderance to folks adopting Avo. Just a thought 😀

**Comment by adrianthedev** [comment]
Hey, thanks for writing to us. We're happy to accommodate different licenses types. 
Initially, we had a per-year unlimited license. We might bring it back.
What is your current use case?

**Reply by brandonzylstra** [reply]
I can think of several use cases: 

1. an agency that does lots of Rails apps across its many clients
2. a university that has lots of internal apps based on Rails
3. a corporation that likewise has lots of internal Rails apps
4. (if it's a lifetime license rather than per-year) a serial entrepreneur who use Avo on every new project

**Reply by viswanathanb** [reply]
Without understanding the internals ( yet ) , single single site will run into scalability issues for a large muti tenant application. With multiple sites, there are other ways to address scalability ( w.r.t multiple databases , etc. ) . 

With the current design ( even other alternatives like Forestadmin ) forces certain opinionated way of development.  

A multisite affordable pro license is definitely required for startups with multi tenantcy.
